### PR TITLE
edited files

### PR DESCRIPTION
**Pull request by anyanime**
I changed the email and SMS message. Please kindly go through it and give your feedback. Thanks.

### CODE REVIEW

File: cron.js [hunk 1]
```diff
@@ -2,9 +2,9 @@ const mainscript = require("./index.js")
 const CronJob = require('cron').CronJob
 
 console.log('Before job instantiation')
-const job = new CronJob('0/2 * * * * *', function () {
+const job = new CronJob('0/5 * * * * *', function () {
```
Comment (stbensonimoh) [inline cron.js:5]: What does '0/5' mean?

File: index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 "use strict"
 const fetch = require('node-fetch')
 const phone = '+2349022631479'
-const smsMessage = 'Good day, This is to inform you that the light unit in the office will finish in the next 24hours. Kindly send money to benson to load the light unit. Thanks In anticipation of your response. -AWLOLECTRIC BOT.'
+const smsMessage = 'Oops AWLO office is running out of light units! There will be power-outage in 24 hours. Load more units now.  -AWLOLECTRIC BOT.'
```
Comment (stbensonimoh) [inline index.js:4]: This message is great.

**Review by stbensonimoh — APPROVED** [review]
It works for me.

File: cron.js (hunk 1 above)
Comment (anyanime) [reply cron.js:5]: It stands for five seconds. I used it for testing.

**Pull request merged**